src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -67,9 +69,11 @@ public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFact
         public override void LogFunctionExecutionEvent(string executionId, string siteName, int concurrency, string functionName,
             string invocationId, string executionStage, long executionTimeSpan, bool success)
         {
-            var logger = _loggerFactory.GetOrCreate(FunctionsExecutionEventsCategory);
+            bool fastLinuxLoggingEnabled = !string.IsNullOrEmpty(_functionsHostingConfiguration.GetValue(ScriptConstants.HostingConfigEnableLinuxEPFastLog));
+            var logger = _loggerFactory.GetOrCreate(FunctionsExecutionEventsCategory, fastLinuxLoggingEnabled);
             string currentUtcTime = DateTime.UtcNow.ToString();
-            if (FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableLinuxEPExecutionCount))
```

Comment:
So this was previously mistakenly made a feature flag? I agree that it wouldn't make sense for a user to configure this flag.

Reply:
I was not aware of HostingConfig flags until the demo. When FunctionsHost started emitting FunctionExecutionCount logs with fields, it caused a memory leak in the metering service.  To disable it, I used the FeatureFlags.  HostingConfig flags will be much better for toggling the change if it causes any further issues.